packages/api-reference/src/components/ApiClientModal.vue

```diff
@@ -25,6 +26,7 @@ onMounted(async () => {
   const { app, open, updateAuth, modalState, updateSpec, updateServer } =
     await createApiClientModal(el.value, {
       spec: props.spec ?? {},
+      initialScheme: props.preferredSecurityScheme,
```

Comment:
Should this have the same name `preferredSecurityScheme` as in the API Reference configuration?

Reply:
Yea I had it as preferred before but I found it a bit of a misnomer. I named it initial in the client because it won't get updated on change, it only gets set on import